src/commands/install.ts

```diff
@@ -108,10 +217,30 @@ export async function install(
       process.chdir(cwd);
     }
 
-    const ruleCollection = initRuleCollection();
+    // Handle monorepo mode first, before checking for config
+    if (options.monorepo) {
```

Comment:
Might have missed, but does this mean you infer the project is a monorepo? how is options populated? why isn't only the config?